pkg/cluster/meta/topology.go

```diff
@@ -293,13 +292,15 @@ func (s PDSpec) Status(pdList ...string) string {
 		suffix = "|UI"
 	}
 
-	healths, err := curPdAPI.GetHealth()
+	// "Down" means all PD nodes don't work,
+	// while "Unhealthy" means current PD node doesn't work
```

Comment:
I'm confused by this, I thought "Down" means the specific PD node is not working, and "Unhealthy" means the PD cluster is not in proper state.

Reply:
But according to the old logic, they will be all recognized as "Down" status, we never can go to the `return "Unhealthy" + suffix` line logic.